packages/fuels-contract/src/script.rs

```diff
@@ -35,6 +36,7 @@ impl Script {
     // modified, it is only simulated using a "dry-run".
     pub async fn simulate(self, fuel_client: &FuelClient) -> Result<Vec<Receipt>, Error> {
         let receipts = fuel_client.dry_run(&self.tx).await?;
+        debug!(target: "receipts", "{:?}", receipts);
```

Comment:
The debug statements of receipts should be in the same place, see my suggestion above

Reply:
remove this line to avoid logging the receipts twice